examples/eks_argo/argo_events.tf

```diff
@@ -1,5 +1,5 @@
 module "argo_events" {
   depends_on     = [helm_release.argo]
-  source         = "./argo_events"
+  source         = "git::git@github.com:outerbounds/metaflow-tools//common/terraform/argo_events?ref=v2.0.0"
```

Comment:
Doesn't this mean we need to cut a release for `metaflow-tools`? (I couldn't find `v2.0.0`).

Assuming you are cutting a release before merging this PR.

Reply:
That's right.  I did it a few mins ago.  All ready to go.